src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalWcfTest.cs

```diff
@@ -290,12 +290,7 @@ public static bool NTLM_Available()
         // Returns 'true' if SSL is available to use.
         public static bool SSL_Available()
         {
-            // The current heuristic is that Windows test clients have been
-            // properly configured, otherwise we don't know.  CI and lab runs
-            // will explicitly set this if they have been able to configure
-            // non-Windows test client machines appropriately.
-            return GetConditionValue(nameof(SSL_Available),
-                                     ConditionalTestDetectors.IsWindows);
+           return true;
```

Comment:
Set SSL_Available to true is for testing purpose only. It will not be checked in.